src/app/_components/SiteLayout.tsx

```diff
@@ -14,7 +11,6 @@ export function SiteLayout({ children }: LayoutProps) {
       <body className="m-auto flex max-w-[1032px] flex-col justify-between bg-brand-800 px-6 pb-6 pt-8 text-brand-100">
         <NetlifyIdentityManager />
 
```

Comment:
This was supposed to be a bit of a fallback - Do we have to remove this as to not have duplicates? Did we check if there's a way to keep this as a fallback without having duplicates?

Reply:
Hmm I didn't find a way not have duplicates in the pages if we include that one. And I don't see anything [mentioned in the documentation.](https://nextjs.org/docs/app/building-your-application/optimizing/metadata#json-ld). We could included the checks in cypress test?

Reply:
Yes - We can include checks. Let's add a ticket to the backlog. 🙏🏼